angular.json

```diff
@@ -32,7 +32,8 @@
             ],
             "styles": [
               "./node_modules/@angular/material/prebuilt-themes/deeppurple-amber.css",
-              "src/styles.scss"
+              "src/styles.scss",
+              "./node_modules/material-design-icons-iconfont/dist/material-design-icons.css"
```

Comment:
this adds the material design icons to the project styles so they can be used with <mat-icon>icon_name</mat-icon>